### DISCUSSION TITLE
Is there any alternative way to receive update for Ubuntu user?

### DISCUSSION THREAD

**Discussion by Tim-Cao** [jesseduffield/lazygit#1717] [General]
According to https://github.com/jesseduffield/lazygit#ubuntu, lazygit tool for Ubuntu is deprecated. Is there any alternative way to receive update for Ubuntu user?

**Comment by hiddeninthesand** [comment]
You may find [the MPR package](https://mpr.makedeb.org/packages/lazygit-bin) helpful.

**Comment by ShortCipher** [comment]
I just downloaded the latest binary from:
https://github.com/jesseduffield/lazygit/releases

My Ubuntu machine is 64-bit, so I grabbed:
https://github.com/jesseduffield/lazygit/releases/download/v0.33/lazygit_0.33_Linux_x86_64.tar.gz

Then I unzipped it: 
`tar xzf lazygit_0.33_Linux_x86_64.tar.gz`

And replaced the old binary:
`sudo cp lazygit /usr/bin/`

Quick and dirty. The package manager still thinks I'm running the old version, but who cares since he's not putting out Ubuntu packages anymore. Just repeat these steps to upgrade again.
